### PR TITLE
Update libsawtooth dependency to 0.5

### PR DESCRIPTION
**Pull request by ltseeley**
Updates the libsawtooth dependency to 0.5

### CODE REVIEW

File: validator/src/batch_ffi.rs [hunk 1]
```diff
@@ -26,10 +28,9 @@ impl From<Batch> for PyObjectWrapper {
         let gil = Python::acquire_gil();
         let py = gil.python();
 
-        let batch_proto = protos::batch::Batch::from(native_batch);
         let batch_pb2 = py
             .import("sawtooth_validator.protobuf.batch_pb2")
-            .expect("unable for python to import sawtooth_validator.protobuf.batch_pb2");
+            .expect("Unable to import block_pb2");
```
Comment (agunde406) [inline validator/src/batch_ffi.rs:33]: This change doesn't look intended

File: validator/src/journal/commit_store_ffi.rs [hunk 2]
```diff
@@ -19,13 +19,15 @@ use std::mem;
...
 use std::slice;
 
-use protobuf;
 use sawtooth::journal::commit_store::{
     ByHeightDirection, CommitStore, CommitStoreByHeightIterator,
 };
-use sawtooth::{batch::Batch, block::Block, transaction::Transaction};
-use transact::database::error::DatabaseError;
-use transact::database::lmdb::LmdbDatabase;
+use sawtooth::transact::database::error::DatabaseError;
```
Comment (agunde406) [inline validator/src/journal/commit_store_ffi.rs:25]: This is missing pulling in Transaction and Batch from transact.

Comment (ltseeley) [reply validator/src/journal/commit_store_ffi.rs:25]: Fixed

File: validator/src/batch_ffi.rs (hunk 1 above)
Comment (ltseeley) [reply validator/src/batch_ffi.rs]: Fixed

**Pull request merged**